net/databag/files/databag.init

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh /etc/rc.common
+ 
+USE_PROCD=1
+START=50
+STOP=51
+
+
+config_databag() {
+	config_get store_path "$1" store_path "/tmp/databag/store"
+	config_get service_port "$1" service_port "7001"
+}
+
+start_service() {
+
+	# load config
+	config_load databag
+	config_foreach config_databag databag
+
+	# init store
+	mkdir -p "${store_path}"
+	cp -n /usr/share/databag/databag.db ${store_path}
+
+	# setup serice
+	procd_open_instance 
+	procd_set_param command /usr/bin/databag -p ${service_port} -s ${store_path} -w /usr/share/databag/web/build
```

Comment:
```suggestion

start_service() {
	# load config
	config_load databag
	config_get store_path "config" "store_path" "/tmp/databag/store"
	config_get service_port "config" "service_port" "7001"

	# init store
	mkdir -p "${store_path}"
	cp -n "/usr/share/databag/databag.db" "${store_path}"

	# setup serice
	procd_open_instance 
	procd_set_param command "/usr/bin/databag"
	procd_append_param command -p "${service_port}"
	procd_append_param command -s "${store_path}"
	procd_append_param command -w "/usr/share/databag/web/build"
```

looks like only single instance is considered, so you can simply use named section instead of `config_foreach`.
use `procd_append_param` to make lines shorter and clearer.